pkg/proxy/winkernel/proxier.go

```diff
@@ -841,7 +830,9 @@ func deleteAllHnsLoadBalancerPolicy() {
 		return
 	}
 	for _, plist := range plists {
-		LogJson("policyList", plist, "Remove Policy", 3)
+		if jsonString, err := json.Marshal(plist); err == nil {
+			klog.V(3).InfoS("Remove policy", "policyList", jsonString)
```

Comment:
FYI, `InfoS` already handle []byte display in [klog/pull/237](https://github.com/kubernetes/klog/pull/237). So we don't need to string().